int16 x_min 
int16 x_max
int16 y_min
int16 y_max
int16 cls
float64 height_factor
float64[] probability_distribution
